static/css/place/place.scss

```diff
@@ -198,11 +198,6 @@ h3 {
   padding: 0.25rem;
 }
 
-#map-container {
```

Comment:
does this need to be converted to .map-container instead of deleting?

Reply:
Updated to class